math/MAGICHF.cpp

```diff
@@ -0,0 +1,23 @@
+#include<iostream>
+#include<bits/stdc++.h>
```

Comment:
Please remove this header because it is quite linux specific and increases the compile time